### ISSUE TITLE
Use `about:blank` instead of `chrome://terms` for bf-cache gatherer

### ISSUE THREAD

**Issue by connorjclark** [GoogleChrome/lighthouse#14665]
Marking for 11.0 not because it is a breaking change, but to keep on our radar to check on "sometime in the future".

**Comment by adamraine** [comment]
Using `about:blank` as the temp page still adds an extra "Internal error" when testing bfcache. I'll ping bfcache folks about this again.

@tunetheweb `chrome://blank` isn't a valid page unfortunately. Do you know any other `chrome://xyz` pages that might be less confusing to users but still avoid the bugs we see with `about:blank`.

**Comment by tunetheweb** [comment]
Yeah I was suggesting getting the Chrome team to add that page. Not sure of any that would be less confusing to be honest.

**Comment by tunetheweb** [comment]
`chrome://print/` maybe?

**Comment by ghnp5** [comment]
Coming from #15223,
I was now trying to search about why do I see that page on every test! Now I understand :)
It's indeed confusing to us.

**Comment by pladaria** [comment]
Is this affecting the LCP metric? My page loads in ~1s but the LCP reports ~10s

**Comment by tunetheweb** [comment]
No, LCP is measured as a separate audit before this one.
